analysis/study_definition.py

```diff
@@ -449,6 +451,7 @@
                                         "2": 0.1
                                         }
                                     },
+                                "incidence": 1.0,
                                 },
         bp_sys=patients.mean_recorded_value(
```

Comment:
I think this will need to be moved out of the `bp` variable if you want to use it in the `bp_ht` variable when running on real data

Reply:
Done!